scripts/populate-alpha-and-delta-column/index-tests.js

```diff
@@ -0,0 +1,157 @@
+const chai = require('chai');
+const sinon = require('sinon');
+const sinonChai = require("sinon-chai");
+chai.use(sinonChai);
+const expect = chai.expect;
+const AirtableRecord = require('airtable').Record;
+const _ = require('lodash');
+const { matchData, findAirtableIds, updateRecords } = require('.');
+
+describe.only('Populate alpha and delta column', () => {
```

Comment:
Les checks de la PR sont passés ?

```suggestion
describe('Populate alpha and delta column', () => {
```

Reply:
Bien vu, Les test sont ceux des scripts, je ne pense pas que la CI les joue.